packages/forma-36-react-components/src/components/CopyButton/CopyButton.css

```diff
@@ -12,17 +13,26 @@
   height: calc(1rem * (40 / var(--font-base-default)));
   width: calc(1rem * (40 / var(--font-base-default)));
   border: 1px solid var(--color-element-mid);
-  background: var(--color-element-light);
+  border-radius: var(--border-radius-medium);
```

Comment:
Not sure about this, the border radius here could mess up styles for consumer components, a.k.a UI extensions

Reply:
@mshaaban0 thx for the feedback. Do you suggest to go with `border-radius-small` or to not touch it at all? @fabe maybe you would have some suggestions?

Reply:
a flag/prop to switch to the new version maybe?

Reply:
@mshaaban0 ok, I think let's keep the border-radius 0 for now. on our end, we almost never use this component separately from `TextInput`. Sounds good?

Reply:
@mshaaban0 great! I rolled back the change for border-radius, if you have a moment, please have a look if this would work. 🙂